Alloy/commands/compile/parsers/Ti.UI.NavigationWindow.js

```diff
@@ -4,15 +4,19 @@ var _ = require('lodash'),
 	CU = require('../compilerUtils'),
 	tiapp = require('../../../tiapp');
 
-var MIN_VERSION = '3.1.3';
+const MIN_VERSION = '7.4.0';
 
 exports.parse = function(node, state) {
 	return require('./base').parse(node, state, parse);
 };
 
 function parse(node, state, args) {
 	if (tiapp.version.lt(tiapp.getSdkVersion(), MIN_VERSION)) {
-		U.die('Ti.UI.iOS.NavigationWindow (line ' + node.lineNumber + ') requires Titanium 3.1.3+');
```

Comment:
Ti.UI.iOS.NavigationWindow works in SDK 3.1.3+, but Ti.UI.NavigationWindow in 7.4.0+. Actually 7.5.0, because all non-iOS 12-tickets will be moved from 7.4.0 to 7.5.0. Thanks for this @feons :)